Community/iot_device_registration/README.md

```diff
@@ -41,8 +42,20 @@ organisation Network Access Control using Aruba Clearpass.
 ## Detailed Information
 - The workflow contains a configuration file "iotipallocation.conf" that has to be configured prior to using it, the configuration includes BAM's connectivity, API User credentials, the block and network to use for the DHCP Reserved allocation, the zone for creation of the device name in DNS, email server settings for notifications and other information necessary for the workflow to workflow
 - The workflow has a UI to get the username, email address and the list of IoT Devices to be registered in CSV format
-- Format of the CSV file: name,macAddress,department,phone,location,period
+- Format of the CSV file: name,macAddress,department,phone,location,period  
 
+Registered IoT Device Information  
+![IoT Device Registration - Registered Device](https://i.ibb.co/KxvLW3j/Screenshot-2019-01-21-16-47-47.png)  
+
+Email Notification  
+![IoT Device Registration - Sample Email](https://i.ibb.co/KxvLW3j/Screenshot-2019-01-21-16-47-47.png)  
```

Comment:
Sample email image URL is the same as the Registered Device URL.

Reply:
Hi Delme, I changed the image with with a Setup Diagram. Please kindly review. Thanks.